tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowsersecuritychange_event.html

```diff
@@ -0,0 +1,49 @@
+<head>
```

Comment:
Let's give this a `<title>` and update the ini to use that instead of `Untitled`.